tests/e2e/lib.sh

```diff
@@ -471,21 +488,23 @@ wait_for_collectors_to_be_operational() {
     done
 }
 
+# shellcheck disable=SC2120
 patch_resources_for_test() {
+    local central_namespace=${1:-stackrox}
     info "Patch the loadbalancer and netpol resources for endpoints test"
 
     require_environment "TEST_ROOT"
     require_environment "API_HOSTNAME"
 
-    kubectl -n stackrox patch svc central-loadbalancer --patch "$(cat "$TEST_ROOT"/tests/e2e/yaml/endpoints-test-lb-patch.yaml)"
-    kubectl -n stackrox apply -f "$TEST_ROOT/tests/e2e/yaml/endpoints-test-netpol.yaml"
+    kubectl -n "${central_namespace}" patch svc central-loadbalancer --patch "$(cat "$TEST_ROOT"/tests/e2e/yaml/endpoints-test-lb-patch.yaml)"
+    kubectl -n "${central_namespace}" apply -f "$TEST_ROOT/tests/e2e/yaml/endpoints-test-netpol.yaml"
```

Comment:
This file specifies `meta.namespace`.

Reply:
Sorry, can you elaborate? Not sure what you mean.

Reply:
https://github.com/stackrox/stackrox/blob/master/tests/e2e/yaml/endpoints-test-netpol.yaml#L5
-> this would conflict if the `${central_namespace}` is set to something different than `stackrox`.

Reply:
Yes, but should also be work to remove the `metadata.namespace` field from the file.

Reply:
of course...

Reply:
removed.